test/unit/lodash.spec.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function()
+{
+
+    describe('constant', function()
+    {
+        var params = datasets.constant();
+        it('should check if first parameter is an object',function()
+        {
+            expect(params[0]).to.be.an('object');
+        });
+        it('should check objects values',function()
+        {
+            expect(params[0]).to.have.property('fred');
+            expect(params[0]).to.have.property('pebbles');
+            expect(params[0]).to.have.deep.property('john.age', 36);
+        });
+        it('should check if function returns an object', function(){
+             var getter = _.constant(params[0]);
+             expect(getter()).to.be.an('object');
```

Comment:
Here is good example how to test result of lodash function. Make the same with returned values (now you check only returned type)